app/models/vulnerability.rb

```diff
@@ -29,18 +30,19 @@ class Vulnerability < ActiveRecord::Base
   # Add some fields into this vulnerability object given an `obj` with the
   # `sc` scanner response.
   def add_extra_values!(obj:, sc:)
-    attrs = fetch_attributes(obj: obj, sc: sc)
+    attrs = fetch_attributes(obj: obj)
+    attrs[:scanner] = sc if scanner.blank?
```

Comment:
Moved it here because otherwise rubocop was complaining on the complexity of `fetch_attributes`.